packages/modeling/src/geometries/geom2/applyTransforms.js

```diff
@@ -2,25 +2,26 @@ const mat4 = require('../../maths/mat4')
 const vec2 = require('../../maths/vec2')
 
 /*
- * Apply the transforms of the given geometry.
- * NOTE: This function must be called BEFORE exposing any data. See toSides().
+ * Apply the transforms of the given geometry and return the sides.
```

Comment:
I think the old comments are correct now.

Reply:
I have commited the code that passes tests, to break-up the changes into smaller steps. 
If you agree, I would go even further and remove applyTransforms as they are only used in toSides, toPolygons, toPoints.